PyISY/Nodes/group.py

```diff
@@ -23,7 +23,7 @@ class Group(object):
     status = Property(0)
     hasChildren = False
 
-    def __init__(self, parent, nid, name, members=None, controllers=None):
+    def __init__(self, parent, nid, name, members=None, controllers=None, notes=False):
```

Comment:
I made it false because it requires a separate REST call to grab the notes, so when you call the spoken method and notes is False, it makes the REST call, otherwise it returns the notes.  I was thinking that notes=None meant that it did not have any notes, but I think they way it's written notes="" when there are no notes.